messaging/messaging-core/src/test/java/uk/gov/justice/services/messaging/JsonObjectsTest.java

```diff
@@ -299,4 +303,58 @@ public void shouldCreateBuilderFromJsonObjectWithFilter() {
         assertThat(actual.getString("name"), equalTo(source.getString("name")));
     }
 
+    @Test
+    public void shouldConvertCollectionOfJsonObjectsToJsonArray() {
+
+        final String oldKey = "oldKey";
+        final String newKey = "newKey";
+
+        JsonArray input = Json.createArrayBuilder()
```

Comment:
Example usage should go in the Javadoc or on the Wiki; I'd prefer the tests to just test the functionality.